task3.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+
+def count_files_by_extension(directory_path, extension):
+    try:
+        count = 0
+        for root, dirs, files in os.walk(directory_path):
```

Comment:
```suggestion
        for _, _, files in os.walk(directory_path):
```

Reply:
We can ignore unused positional arguments in python using '_' instead. Everything else looks good.

Reply:
Agreed